argoverse/visualization/visualization_utils.py

```diff
@@ -219,7 +218,7 @@ def show_image_with_boxes(img: np.ndarray, objects: Iterable[ObjectLabelRecord],
         if obj.occlusion == 100:
             continue
         box3d_pts_3d = obj.as_3d_bbox()
-        uv = calib.project_ego_to_image(box3d_pts_3d)
+        calib.project_ego_to_image(box3d_pts_3d)
```

Comment:
can be removed altogether, since `uv` is not used

Reply:
Done 👍